### PR TITLE
[MRG] 06 - unit testing

### PR DESCRIPTION
**Pull request by glemaitre**

### CODE REVIEW

File: Day_2_Software_engineering_best_practices/solutions/06_unit_tests/spectra_analysis.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+"""
```
Comment (jorisvandenbossche) [inline Day_2_Software_engineering_best_practices/solutions/06_unit_tests/spectra_analysis.py]: you can remove this file (We should *maybe* also remove it in the modules solution, I started with this as one module, and only afterwards created the package with the same name.

**Comment by glemaitre** [comment]
done

File: Day_2_Software_engineering_best_practices/solutions/06_unit_tests/spectra_analysis/tests/test_preprocessing.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from os.path import dirname, join
+import numpy as np
+from numpy.testing import (assert_allclose, assert_array_equal,
+                           assert_raises_regex)
+from ..preprocessing import read_spectra
```
Comment (jorisvandenbossche) [inline Day_2_Software_engineering_best_practices/solutions/06_unit_tests/spectra_analysis/tests/test_preprocessing.py]: for testing I think we should use the absolute path

Comment (glemaitre) [reply Day_2_Software_engineering_best_practices/solutions/06_unit_tests/spectra_analysis/tests/test_preprocessing.py]: ups

**Pull request merged**